README.md

```diff
@@ -10,6 +10,15 @@
 `pynxtools` is a tool designed for making your experimental data FAIR.
 It allows to develop ontologies and to create ontological instances based on the [NeXus format](https://www.nexusformat.org/).
 
+# Scope
+
+`pynxtools` (previously called `nexusutils`) is intended as a parser for combining various instrument output formats and electronic lab notebook (ELN) formats to an hdf5 file according to NeXus application definitions.
```

Comment:
add until which version

Reply:
eventually introduce hdf5 upon first usage and add link to the hdf5 consortium page,

"is a parser" instead of "is intended"

Reply:
I think we can actually just remove the part about "previously called `nexusutils`". It was only called like this until v0.0.1 and that one was not working anyway.

Rest was added.